src/Sidekick/Helpers/Input/EventsHandler.cs

```diff
@@ -31,9 +31,22 @@ public static void Initialize()
             Legacy.KeybindEvents.OnLeaveParty += TriggerLeaveParty;
             Legacy.KeybindEvents.OnOpenSearch += TriggerOpenSearch;
             Legacy.KeybindEvents.OnOpenLeagueOverview += TriggerLeagueOverlay;
+            Legacy.KeybindEvents.OnWhisperReply += TriggerReplyToLatestWhisper;
             Legacy.KeybindEvents.OnMouseClick += MouseClicked;
         }
 
+        private static Task TriggerReplyToLatestWhisper()
+        {
+            var characterName = Legacy.WhisperService.GetLatestWhisperCharacterName();
+            if(!string.IsNullOrEmpty(characterName))
+            {
+                Legacy.NativeClipboard.SetText(string.Empty);
+                Legacy.NativeClipboard.SetText($"@{characterName} ");
+                Legacy.NativeKeyboard.SendCommand(KeyboardCommandEnum.ReplyToLatestWhisper);
```

Comment:
We will need to find a way to write directly inside the chat window without having to do clipboard manipulation.
This is not possible at the moment without changing the NativeKeyboard. Let me know if you want to make this change.

Reply:
It feels really clunky if we do it like in the return to hideout command. Using the clipboard makes this instant.